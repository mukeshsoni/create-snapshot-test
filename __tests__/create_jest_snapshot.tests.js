const createJestSnapshot = require("../src/jest_create_snapshot")
const path = require("path")

it("should pass", () => {
  const fakeComponentPath = path.join(__dirname + "/../fake_component.js")
  expect(createJestSnapshot(fakeComponentPath)).toMatchSnapshot()
})
