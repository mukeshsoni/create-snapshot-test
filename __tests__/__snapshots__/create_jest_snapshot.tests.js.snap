// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should pass 1`] = `
"import React from 'react';
import renderer from 'react-test-renderer'
import FakeComponent from \\"/Users/mukesh/Documents/projects/jest-create-snapshot/fake_component.js\\"

test('Should render FakeComponent correctly', () => {
  const tree = renderer
    .create(<FakeComponent firstName={\\"firstName\\"} person={{\\"firstName\\":\\"person.firstName\\"}} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
"
`;
